voidcrew/code/modules/overmap/ships/pvp/misc.dm

```diff
@@ -0,0 +1,55 @@
+#define SHIP_GRACE_TIME (1 MINUTES)
```

Comment:
make sure to `#undef` this at the end of the file (i assume it's used nowhere else)